pkg/scaffold/manager/cmd.go

```diff
@@ -70,12 +71,17 @@ func main() {
 
 	// Setup Scheme for all resources
 	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
-		log.Fatal(err)
+		log.Fatalf("Add apis failed, error: %#v.", err)
 	}
 
 	// Setup all Controllers
 	if err := controller.AddToManager(mgr); err != nil {
-		log.Fatal(err)
+		switch err.(type) {
+		case *meta.NoKindMatchError:
+			log.Fatalf("No kind %v found, please install CRD in advance.", err.(*meta.NoKindMatchError).GroupKind)
```

Comment:
I am wondering if we should add more context in the error in the `controller-runtime` library itself instead of putting the logic in generated code, WDYT ?

Reply:
My thoughts:
Generally, lib (controller-tools) emits all errors and client (kubebuilder) decides what behavior to perform. Based on this point, my PR make sense.

But considering our client code is generated, more concise the better. I think to handle this error in controller-tools is also a good idea. Do you have some ideas about which code should I modify?

Thanks!

Reply:
@droot just a ping, any thoughts?

Reply:
Sorry, didn't follow this thread.

Actually, our client code is no longer generated. Kubebuilder project uses client pkg from https://github.com/kubernetes-sigs/controller-runtime. May be, we can just wrap the error when returning from client's pkg. Can you pl. investigate.

Reply:
Sure, I will check later. 
I will ping you if have some problems. Thanks.